src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -287,6 +282,17 @@ private void RemovePlaceholderReplacement(BoundValuePlaceholderBase placeholder)
             Debug.Assert(removed);
         }
 
+        /// <summary>
+        /// Remove all the listed placeholders.
+        /// </summary>
+        private void RemovePlaceholderReplacements(IEnumerable<BoundValuePlaceholderBase> placeholders)
```

Comment:
I suspect in the future it will be handy again to be able to remove multiple placeholders. I'm not sure what is the downside of using `IEnumerable` to keep options open.